getting_started/topics/first-boot/distribution-files-product.adoc

```diff
@@ -1,24 +1,31 @@
 
-=== Installing the Server
+[id="standalone-install"]
+=== Installing the {project_name} server
 
-Download the {project_name} Server:
+For this sample instance of {project_name}, this procedure involves installation in standalone mode. The server download ZIP file contains the scripts and binaries to run the {project_name} server. You can install the server on Linux or Windows.
 
-* *rh-sso-{project_version}.zip*
 
-NOTE: This file can be downloaded from https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[the Red Hat customer portal].
+.Procedure
 
-The *rh-sso-{project_version}.zip* file is the server-only distribution. It contains only the scripts and binaries to run the {project_name} server.
+. Go to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[Red Hat customer portal].
 
-Place the file in a directory you choose and use the `unzip` utility to unpack it, like this:
+. Download the {project_name} Server: *rh-sso-{project_version}.zip*
 
+. Place the file in a directory you choose.
+
+. Unpack the ZIP file using the appropriate `unzip` utility.
+
++
 .Linux/Unix
 [source,bash,subs=+attributes]
 ----
 $ jar xf rh-sso-{project_version}.zip
```

Comment:
@andymunro This section: lines 14 - 22 should be unified with: distribution-files-community.adoc 
Here is used the jar command and in the community file is used the tar command. Also this description contains unzip command only and the community file the jar and unzip commands instead. I propose to write there unzip, tar, jar and use unzip as an example.

Reply:
@lhanusov Are you suggesting that I use this format for the example

unzip rhsso-{project-version}.zip (for product)
AND
unzip keycloak-{project_version}.zip  (for community)

If so, I would not need the headers Windows/Linux etc.   I believe unzip works on both platforms.

Reply:
Well, unzip command is by default on the linux machine, but on the windows machine there is no unzip command as the default package. @vmuzikar @Pepo48 Vasku and Peto, please can you help us which is the first choice cmd command for unzip? Because normally I will use 7zip or something similar via GUI. I found that jar xf command should definitely work, but it is connected with the java installation.

Reply:
@lhanusov Not sure, @Pepo48 is the Win master here. :) Maybe [`Expand-Archive`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/expand-archive?view=powershell-7) is the correct Windows way?

Reply:
@lhanusov, as @vmuzikar has already mentioned, currently the most Windows-like way of extracting an archive is to use PowerShell CmdLet `Expand-Archive`, you can then use it as following:  
```Expand-Archive -Path 'C:\input.zip' -DestinationPath 'C:\output'```